inputs/orchestrator_inner:6.json

```diff
@@ -59,12 +42,7 @@
   ],
   "buildstep_plugins": [
     {
-      "name": "orchestrate_build",
-      "args": {
-        "platforms": "{{PLATFORMS}}",
-        "build_kwargs": "{{BUILD_KWARGS}}",
-        "osbs_client_config": "{{OSBS_CLIENT_CONFIG}}"
-      }
+      "name": "orchestrate_build"
     }
   ],
   "postbuild_plugins": [
```

Comment:
I can't comment further down, but let's also remove:

- [ ] `dockpulp_loglevel` from `pulp_sync` and `pulp_publish`
- [ ]  `"koji_keytab"` and  `"koji_principal"` from `koji_tag_build`
- [ ] `"verify_ssl"` and "build_json_dir" from `import_image`
- [ ] `"verify_ssl"` from `store_metadata_in_osv3`

I think it's also safe to make `remove_worker_metadata` required by remove `"required": false` property.

Reply:
I'm not sure what to make out of `pulp_pull`, maybe that can be addressed later.

Reply:
Fixed I think.